app/src/main/java/org/mozilla/fenix/theme/FirefoxTheme.kt

```diff
@@ -15,6 +15,7 @@ import androidx.compose.runtime.remember
 import androidx.compose.runtime.setValue
 import androidx.compose.runtime.staticCompositionLocalOf
 import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.platform.LocalContext
```

Comment:
This looks unused to me, and I would probably skip the new line removal below. My reasoning for this is that we should utilize new lines to separate variable declarations from bigger functional blocks like `ProvideFirefoxColors(colors) {}`.